File: fizzbuzz/fizzbuzz.py
#!/usr/bin/env python3

for x in range(0,100):
  if x%15 == 0:
    print("fizzbuzz")
  elif x%3 == 0:
    print("fizz")
  elif x%5 == 0:
    print("buzz")
  else:
    print(x)

File: fizzbuzz/fizzbuzz.swift
// shebang??
let number = [1,2,3,4,5,6,7,8,9,10]

for i in 0...100 {
  if i % 3 == 0 && i % 5 == 0 {
    print("fizzbuzz")
  } else if  i % 3 == 0 {
    print("fizz")
  } else if i % 5 == 0 {
    print("buzz")
  } else {
    print("\(i)")
  }
}